Changelog.md

```diff
@@ -22,6 +22,10 @@
 1. Bump in edifice: ign-rendering5
     * [Pull request #55](https://github.com/ignitionrobotics/ign-sensors/pull/55)
 
+1. Documentation updates
+    * [Pull request #116](https://github.com/ignitionrobotics/ign-sensors/pull/116)
+    * [Pull request #114](https://github.com/ignitionrobotics/ign-sensors/pull/114)
```

Comment:
this is one more of a bugfix / revert than doc update.

Reply:
Oops, see https://github.com/ignitionrobotics/ign-sensors/pull/126

Reply:
Updated this PR